tests/integration/test_scan.py

```diff
@@ -339,6 +339,8 @@ async def test_workflow_params(
             assert uuid_key in flow['workflow_params']
             # check the workflow uuid key matches the scheduler value
             assert flow['workflow_params'][uuid_key] == schd.uuid_str
+        else:
```

Comment:
Need a `break` in the loop otherwise the `else` block will always run
```suggestion
            break
        else:
```

Reply:
(still looking at this as the situation with xdist is now reversed)